GRAMMAR.tex

```diff
@@ -1314,22 +1314,28 @@ \subsection{Agent and Patient Nouns}% agent noun, patient noun
 
 %% ‘lẹ-’: from PF ‘plus’. Comparative prefix.
 \subsection{Comparison}\label{subsec:comparison}
-Unlike in many other languages, there are 4 comparatives in UF: The affirming comparative, so called
-because it affirms the positive (‘better, and also good’); the denying comparative, which denies the positive
-(‘better, but not good’), the sufficient comparative, which indicates that there is just enough of something,
-and the neutral comparative, which does not make any statement about the positive (‘better’).
+Unlike in many other languages, there are 8 comparatives in UF, 4 positive and 4 negative: The affirming comparatives, so called
+because it affirms the comparison (‘better, and also good’, ‘less hot, and also not very hot’); the denying comparative, which denies the comparison
+(‘better, but not good’, ‘less hot, but still hot’), the sufficient comparative, which indicates that there is just enough of something,
+and the neutral comparative, which does not make any statement about the comparison (‘better’, ‘less hot’).
```

Comment:
```suggestion
Unlike in many other languages, there are 8 comparatives in UF, 4 positive and 4 negative: The affirming comparatives, so called
because they affirms the comparison (‘better, and also good’, ‘less hot, and also not very hot’); the denying comparatives, which deny the comparison
(‘better, but not good’, ‘less hot, but still hot’), the sufficient comparatives, which indicate that there is just enough of something,
and the neutral comparatives, which do not make any statement about the comparison (‘better’, ‘less hot’).
```